app/src/main/java/com/framgia/moviedbtraining/movieDetails/MovieDetailsActivity.kt

```diff
@@ -151,7 +153,16 @@ class MovieDetailsActivity : AppCompatActivity(), MovieDetailsContract.ViewModel
     dialog.show()
   }
 
+  fun openImageFullScreen() {
+    ImageFullActivity.MovieDetailsIntent(mMovie)

```

Comment:
Updated in other pr https://github.com/nvmanh/kotlin-moviedb-training/pull/31/files